db/migrate/20200709160133_set_empty_filter_taxonomy_search_nil.rb

```diff
@@ -0,0 +1,5 @@
+class SetEmptyFilterTaxonomySearchNil < ActiveRecord::Migration[6.0]
+  def change
+    Filter.where(taxonomy_search: '').update(taxonomy_search: nil)
```

Comment:
I believe we would be better off with `update_all` as it doesn't go throught the whole `ActiveRecord` callback hell. It's better to be avoided in migrations.
```suggestion
    Filter.where(taxonomy_search: '').update_all(taxonomy_search: nil)
```